src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -894,6 +929,34 @@ func (c controller) uploadSummaryLogs(podName string, namespace string, sinceSec
 	return nil
 }
 
+func (c controller) parseMustGatherImages() []string {
+	images := make([]string, 0)
+	if c.MustGatherImage == "" {
+		c.log.Infof("collecting must-gather logs into using image from release\n")
+		return images
+	}
+
+	var imageMap map[string]string
+	err := json.Unmarshal([]byte(c.MustGatherImage), &imageMap)
+	c.log.Infof("collecting must-gather logs into using a this image setup %s\n", c.MustGatherImage)
```

Comment:
Something with this log syntax isn't right.

Reply:
In this case the c.MustGatherImage is just a string

Reply:
Actually it is always a string

Reply:
moved up and re-worded